superset-frontend/webpack.config.js

```diff
@@ -110,45 +108,29 @@ if (isDevMode) {
   plugins.push(new OptimizeCSSAssetsPlugin());
 }
 
-const BABEL_JAVASCRIPT_OPTIONS = {
-  presets: [
-    [
-      '@babel/preset-env',
-      {
-        useBuiltIns: 'usage',
-        corejs: 3,
-        loose: true,
-        shippedProposals: true,
-        modules: false,
-        targets: false,
-      },
-    ],
-    '@babel/preset-react',
-  ],
-  plugins: [
-    'lodash',
-    'react-hot-loader/babel',
-    '@babel/plugin-proposal-object-rest-spread',
-    '@babel/plugin-proposal-class-properties',
-    '@babel/plugin-syntax-dynamic-import',
-  ],
-};
-
-const BABEL_TYPESCRIPT_OPTIONS = {
-  presets: BABEL_JAVASCRIPT_OPTIONS.presets.concat([
-    '@babel/preset-typescript',
-  ]),
-  plugins: BABEL_JAVASCRIPT_OPTIONS.plugins.concat([
-    'babel-plugin-typescript-to-proptypes',
-  ]),
-};
-
-const PREAMBLE = ['babel-polyfill', path.join(APP_DIR, '/src/preamble.js')];
+const PREAMBLE = [path.join(APP_DIR, '/src/preamble.js')];
```

Comment:
`babel-polyfill` is deprecated. `@babel/preset-env` and `@babel/plugin-transform-runtime` should take care of already.